### PR TITLE
Combining metrics profiles files

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Want to combine the metrics and metrics-ovn yaml files so the default metrics file has podCPU and podMemory that can be used when creating the benchmark-comparison for kube-burner

### Fixes

1. Needing to set metrics-ovn.yaml file to be able to use all of the touchstone-config files for google sheet comparison generation
2. Remove duplication of files that are very similar

### CODE REVIEW

**Comment by paigerube14** [comment]
@rsevilla87 @dry923 are we able to get this PR merged? The default metrics profile doesn't collect all the data that is available when using all the [touchstone-configs ](https://github.com/cloud-bulldozer/e2e-benchmarking/tree/master/workloads/kube-burner/touchstone-configs) here


```
09-26 09:28:11.949  2022-09-26 13:28:11,548 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'metricName.keyword': 'podCPU', 'labels.namespace.keyword': 'openshift-sdn'}, 'buckets': ['labels.pod.keyword', 'labels.node.keyword'], 'aggregations': {'value': ['avg']}}
09-26 09:28:12.522  Traceback (most recent call last):
09-26 09:28:12.522    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 70, in <module>
09-26 09:28:12.522      main()
09-26 09:28:12.522    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 53, in main
09-26 09:28:12.522      df = read_csv(csv_filepath)
09-26 09:28:12.522    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 35, in read_csv
09-26 09:28:12.522      df = pd.read_csv(filepath)
09-26 09:28:12.522    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 211, in wrapper
09-26 09:28:12.522      return func(*args, **kwargs)
09-26 09:28:12.522    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 317, in wrapper
09-26 09:28:12.522      return func(*args, **kwargs)
09-26 09:28:12.522    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
09-26 09:28:12.522      return _read(filepath_or_buffer, kwds)
09-26 09:28:12.522    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 605, in _read
09-26 09:28:12.522      parser = TextFileReader(filepath_or_buffer, **kwds)
09-26 09:28:12.522    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
09-26 09:28:12.522      self._engine = self._make_engine(f, self.engine)
09-26 09:28:12.522    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1747, in _make_engine
09-26 09:28:12.522      return mapping[engine](f, **self.options)
09-26 09:28:12.522    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 92, in __init__
09-26 09:28:12.522      self._reader = parsers.TextReader(src, **kwds)
09-26 09:28:12.522    File "pandas/_libs/parsers.pyx", line 554, in pandas._libs.parsers.TextReader.__cinit__
09-26 09:28:12.522  pandas.errors.EmptyDataError: No columns to parse from file
09-26 09:28:12.780  [1mMon Sep 26 13:28:12 UTC 2022 Querying results[0m
09-26 09:28:12.780  [1mMon Sep 26 13:28:12 UTC 2022 Running: touchstone_compare --database elasticsearch -url [https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443](https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com/) -u f6ea8e10-7001-4f6a-9c02-967850db998d --config /tmp/podCPU-max.json -o csv --output-file /tmp/cluster-density-f6ea8e10-7001-4f6a-9c02-967850db998d/podCPU-max.json[0m
09-26 09:28:14.163  2022-09-26 13:28:13,768 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'metricName.keyword': 'podCPU', 'labels.namespace.keyword': 'openshift-sdn'}, 'buckets': ['labels.pod.keyword', 'labels.node.keyword'], 'aggregations': {'value': ['max']}}
09-26 09:28:14.732  Traceback (most recent call last):
09-26 09:28:14.732    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 70, in <module>
09-26 09:28:14.732      main()
09-26 09:28:14.732    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 53, in main
09-26 09:28:14.732      df = read_csv(csv_filepath)
09-26 09:28:14.732    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 35, in read_csv
09-26 09:28:14.732      df = pd.read_csv(filepath)
09-26 09:28:14.732    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 211, in wrapper
09-26 09:28:14.732      return func(*args, **kwargs)
09-26 09:28:14.732    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 317, in wrapper
09-26 09:28:14.732      return func(*args, **kwargs)
09-26 09:28:14.732    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
09-26 09:28:14.732      return _read(filepath_or_buffer, kwds)
09-26 09:28:14.732    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 605, in _read
09-26 09:28:14.732      parser = TextFileReader(filepath_or_buffer, **kwds)
09-26 09:28:14.732    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
09-26 09:28:14.732      self._engine = self._make_engine(f, self.engine)
09-26 09:28:14.732    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1747, in _make_engine
09-26 09:28:14.732      return mapping[engine](f, **self.options)
09-26 09:28:14.732    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 92, in __init__
09-26 09:28:14.732      self._reader = parsers.TextReader(src, **kwds)
09-26 09:28:14.732    File "pandas/_libs/parsers.pyx", line 554, in pandas._libs.parsers.TextReader.__cinit__
09-26 09:28:14.732  pandas.errors.EmptyDataError: No columns to parse from file
09-26 09:28:14.732  podMemory-avg.json
09-26 09:28:14.732  [1mMon Sep 26 13:28:14 UTC 2022 Querying results[0m
09-26 09:28:14.732  [1mMon Sep 26 13:28:14 UTC 2022 Running: touchstone_compare --database elasticsearch -url [https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443](https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com/) -u f6ea8e10-7001-4f6a-9c02-967850db998d --config /tmp/podMemory-avg.json -o csv --output-file /tmp/cluster-density-f6ea8e10-7001-4f6a-9c02-967850db998d/podMemory-avg.json[0m
09-26 09:28:16.165  2022-09-26 13:28:15,726 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'metricName.keyword': 'podMemory', 'labels.namespace.keyword': 'openshift-sdn'}, 'buckets': ['labels.pod.keyword', 'labels.node.keyword'], 'aggregations': {'value': ['avg']}}
09-26 09:28:16.730  Traceback (most recent call last):
09-26 09:28:16.730    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 70, in <module>
09-26 09:28:16.730      main()
09-26 09:28:16.730    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 53, in main
09-26 09:28:16.730      df = read_csv(csv_filepath)
09-26 09:28:16.730    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 35, in read_csv
09-26 09:28:16.730      df = pd.read_csv(filepath)
09-26 09:28:16.730    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 211, in wrapper
09-26 09:28:16.730      return func(*args, **kwargs)
09-26 09:28:16.730    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 317, in wrapper
09-26 09:28:16.730      return func(*args, **kwargs)
09-26 09:28:16.730    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
09-26 09:28:16.730      return _read(filepath_or_buffer, kwds)
09-26 09:28:16.730    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 605, in _read
09-26 09:28:16.730      parser = TextFileReader(filepath_or_buffer, **kwds)
09-26 09:28:16.730    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
09-26 09:28:16.730      self._engine = self._make_engine(f, self.engine)
09-26 09:28:16.730    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1747, in _make_engine
09-26 09:28:16.730      return mapping[engine](f, **self.options)
09-26 09:28:16.730    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 92, in __init__
09-26 09:28:16.730      self._reader = parsers.TextReader(src, **kwds)
09-26 09:28:16.730    File "pandas/_libs/parsers.pyx", line 554, in pandas._libs.parsers.TextReader.__cinit__
09-26 09:28:16.730  pandas.errors.EmptyDataError: No columns to parse from file
09-26 09:28:16.730  podMemory-max.json
09-26 09:28:16.730  [1mMon Sep 26 13:28:16 UTC 2022 Querying results[0m
09-26 09:28:16.730  [1mMon Sep 26 13:28:16 UTC 2022 Running: touchstone_compare --database elasticsearch -url [https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443](https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com/) -u f6ea8e10-7001-4f6a-9c02-967850db998d --config /tmp/podMemory-max.json -o csv --output-file /tmp/cluster-density-f6ea8e10-7001-4f6a-9c02-967850db998d/podMemory-max.json[0m
09-26 09:28:17.728  2022-09-26 13:28:17,713 - touchstone - ERROR - Error: Issue capturing results from elasticsearch using config {'filter': {'metricName.keyword': 'podMemory', 'labels.namespace.keyword': 'openshift-sdn'}, 'buckets': ['labels.pod.keyword', 'labels.node.keyword'], 'aggregations': {'value': ['max']}}
09-26 09:28:18.726  Traceback (most recent call last):
09-26 09:28:18.726    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 70, in <module>
09-26 09:28:18.726      main()
09-26 09:28:18.726    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 53, in main
09-26 09:28:18.726      df = read_csv(csv_filepath)
09-26 09:28:18.726    File "/home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/workloads/kube-burner/../../utils/csv_modifier.py", line 35, in read_csv
09-26 09:28:18.726      df = pd.read_csv(filepath)
09-26 09:28:18.726    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 211, in wrapper
09-26 09:28:18.726      return func(*args, **kwargs)
09-26 09:28:18.726    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/util/_decorators.py", line 317, in wrapper
09-26 09:28:18.726      return func(*args, **kwargs)
09-26 09:28:18.726    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
09-26 09:28:18.726      return _read(filepath_or_buffer, kwds)
09-26 09:28:18.726    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 605, in _read
09-26 09:28:18.726      parser = TextFileReader(filepath_or_buffer, **kwds)
09-26 09:28:18.726    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
09-26 09:28:18.726      self._engine = self._make_engine(f, self.engine)
09-26 09:28:18.726    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1747, in _make_engine
09-26 09:28:18.726      return mapping[engine](f, **self.options)
09-26 09:28:18.726    File "/tmp/tmp.sUaaQWVrxv/lib/python3.9/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 92, in __init__
09-26 09:28:18.726      self._reader = parsers.TextReader(src, **kwds)
09-26 09:28:18.726    File "pandas/_libs/parsers.pyx", line 554, in pandas._libs.parsers.TextReader.__cinit__
09-26 09:28:18.726  pandas.errors.EmptyDataError: No columns to parse from file
09-26 09:28:18.726  [1mMon Sep 26 13:28:18 UTC 2022 Installing requirements to generate spreadsheet[0m
09-26 09:28:23.993  WARNING: You are using pip version 21.1.3; however, version 22.2.2 is available.
09-26 09:28:23.993  You should consider upgrading via the '/tmp/tmp.sUaaQWVrxv/bin/python3 -m pip install --upgrade pip' command.
09-26 09:28:23.993  [1mMon Sep 26 13:28:23 UTC 2022 Sheetname: cluster-density-2022-09-26T13:28:23[0m
09-26 09:28:23.993  [1mMon Sep 26 13:28:23 UTC 2022 CSV: /tmp/cluster-density-f6ea8e10-7001-4f6a-9c02-967850db998d/f6ea8e10-7001-4f6a-9c02-967850db998d.csv[0m
09-26 09:28:23.993  [1mMon Sep 26 13:28:23 UTC 2022 Email: sninganu@redhat.com[0m
09-26 09:28:23.993  [1mMon Sep 26 13:28:23 UTC 2022 Service-account: /home/jenkins/ws/workspace/multibranch-pipeline_kube-burner/.gsheet.json[0m
```

**Comment by paigerube14** [comment]
@rsevilla87 @jtaleric @dry923 any thoughts on this? Can we get this merged? 

Screen Shot 2022-09-29 at 10 06 53 AM (2) the only difference in those 2 files is the podCPU and podMemory. 

This small difference is also causing discrepancies in the grafana dashboards in the node section because it’s looking for “podCPU” that isn’t captured in the metrics.yaml file that nodes that aren't for the UUID are being listed 
_uuid.keyword: $uuid AND metricName: "podCPU" AND labels.node.keyword: $master AND labels.namespace.keyword: $namespace_
Screen Shot 2022-09-29 at 10 12 48 AM

**Review by jtaleric — APPROVED** [review]
lgtm - I am however concerned touchstone doesn't do a good job with error handling :(

**Comment by paigerube14** [comment]
Think the issue with touchstone is similar to : https://github.com/cloud-bulldozer/benchmark-comparison/issues/57 not sure if it's the exact same thing though

File: workloads/kube-burner/touchstone-configs/containerMemory-avg.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "elasticsearch": {
+    "ripsaw-kube-burner": [
```
Comment (rsevilla87) [inline workloads/kube-burner/touchstone-configs/containerMemory-avg.json:3]: Why 4 configuration files rather than a single one holding all queries?

**Comment by rsevilla87** [comment]
Hey @paigerube14, I managed to query CPU and memory metrics from pods using the new touchstone config files you've added:

```
(touchstone) rsevilla@wonderland ~/labs/touchstone (master) $ touchstone_compare --database elasticsearch -url nope -u  ab19641c-node-density-20220921   --config kubeburner.json 
+--------------+--------------------------+----------------------+------------------+------------+--------------------------------+
|  metricName  |     labels.namespace     |      labels.pod      |   labels.node    |   metric   | ab19641c-node-density-20220921 |
+--------------+--------------------------+----------------------+------------------+------------+--------------------------------+
| containerCPU | openshift-ovn-kubernetes | ovnkube-master-v4cdp |     master-1     | avg(value) |       5.589392743554736        |
| containerCPU | openshift-ovn-kubernetes | ovnkube-master-xbxhz |     master-2     | avg(value) |       5.230984553671838        |
| containerCPU | openshift-ovn-kubernetes | ovnkube-master-mtdc7 |     master-0     | avg(value) |       1.4631622188757265       |
| containerCPU | openshift-ovn-kubernetes |  ovnkube-node-84rpf  | worker011-5039ms | avg(value) |       1.6327197147460035       |
| containerCPU | openshift-ovn-kubernetes |  ovnkube-node-p2w9h  | worker019-5039ms | avg(value) |       1.5396597782833599       |
| containerCPU | openshift-ovn-kubernetes |  ovnkube-node-fqwvh  |     master-1     | avg(value) |       0.6787404186677101       |
| containerCPU | openshift-ovn-kubernetes |  ovnkube-node-9xwx2  | worker005-5039ms | avg(value) |       1.5002050950309012       |
| containerCPU | openshift-ovn-kubernetes |  ovnkube-node-jg97h  | worker003-5039ms | avg(value) |       1.5401379679754268       |

```

I think we can skip podCPU and podMemory metrics in metrics.yaml

File: workloads/kube-burner/touchstone-configs/containerMemory-avg.json (hunk 1 above)
Comment (jtaleric) [reply workloads/kube-burner/touchstone-configs/containerMemory-avg.json:3]: I sent a example on slack on how to consolidate this.

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**